packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsCompleted/TransactionsCompleted.tsx

```diff
@@ -14,7 +15,8 @@ type TProps = {
 };
 
 const TransactionsCompleted: React.FC<TProps> = ({ filter }) => {
-    const { data, fetchNextPage, isFetching, setFilter } = useTransactions();
+    const { data: transactions, fetchNextPage, isFetching, isLoading, setFilter } = useTransactions();
+    const [shouldShowLoader, setShouldShowLoader] = useState(true);
```

Comment:
```suggestion
    const { data: transactions, fetchNextPage, isFetching, isLoading, setFilter } = useTransactions();
```
There will be some delay if you sync up loading state like this. Best to use derive from state instead of creating new one

```
const shouldShowLoader = !isFetching && !isLoading
```

Reply:
Done